### DISCUSSION TITLE
Paginate without a query string.

### DISCUSSION THREAD

**Discussion by BrendanBasson4** [filamentphp/filament#1342] [Help]
I am working with a list of models and when a different model is selected the pagination persists.<br> 
Is it possible to override this somehow or manually change the page of the table from within the component?<br>
Or explicitly not use it as a query string?

**Comment by danharrin** [comment]
Not sure what the issue is, you have multiple tables on the same page and the pagination is being shared across all tables?

**Comment by BrendanBasson4** [comment]
I have a side bar with clients when a client is switched the page just refreshes the components data without a URL change.<br>
So my problem is if client A is paginated to page 2 and I switch to client B their table is also paginated to page 2.<br> 
So the pagination persist across all clients and does not reset for individual clients.

**Reply by danharrin** [reply]
Livewire's pagination uses the query string to store the page by default, I don't think there's a way around that. Surely its your code's responsibility to ensure that the query string is reset though?

**Reply by BrendanBasson4** [reply]
Ah ok thank you just one last question is it possible to implement my own paginator?

**Comment by tahaRazaTalibi** [comment]
I faced the same issue, I wrote the following code in boot() to use query string identifier for every table (It was not a solution at the time of question):
`
Table::configureUsing(function (Table $table): void {
            $table
                ->queryStringIdentifier(fn($table)=> $table->getModelLabel());
        });
`
